### PR TITLE
Aligning references

### PR DESCRIPTION
**Pull request by ygrange**
I reworded the references in the Supernova table so that they look the same as in the other tables. By doing that I noticed one thing you may want to fix: Some of the references lack page numbers (Gronow et al. 2021 could be two papers for instance). You may want to fix that too.

### CODE REVIEW

**Comment by ygrange** [comment]
This also is connected to https://github.com/openjournals/joss-reviews/issues/4461

File: README.rst [hunk 1]
```diff
@@ -214,7 +214,7 @@ If you find a bug or have a question, please [open an issue in the project's rep
 
 Contributions are welcome, please read our `contributing guidelines`_.
 
-.. _`contributing guidelines`: https://github.com/xuanxu/starmatrix/blob/main/CONTRIBUTING.md
+.. _`contributing guidelines`: https://github.com/xuanxu/starmatrix/blob/review/CONTRIBUTING.md
```
Comment (xuanxu) [inline README.rst:217]:
```suggestion
.. _`contributing guidelines`: https://github.com/xuanxu/starmatrix/blob/main/CONTRIBUTING.md
```

**Review by xuanxu — APPROVED** [review]
:+1:

**Comment by xuanxu** [comment]
Missing page numbers added: https://github.com/xuanxu/starmatrix/commit/2ff4bc04a8b7fcf79e2e3c60e5de41f5918eebe1

**Pull request merged**